api/net/Areas/Subscriber/Controllers/LookupController.cs

```diff
@@ -128,7 +128,7 @@ public IActionResult FindAllAsync()
         var license = _licenseService.FindAll();
         var series = _seriesService.FindAll();
         var ministers = _ministerService.FindAll();
-        var tagServices = _tagService.FindAll();
+        var tagServices = _tagService.FindAllEnabled();
```

Comment:
Don't do this, as it will cause issues when content references a disabled tag.  I would think that we still want to be able to search for content with these old tags.